tests/integration/test_cli_daemon.py

```
@@ -265,3 +265,69 @@ def test_process_exception_exits_1(
+        from daemon.config import DaemonConfig
+        from daemon.service import DaemonService
+
+        daemon = DaemonService(DaemonConfig())
+        r, w = os.pipe()
+        os.set_blocking(r, False)
+        os.set_blocking(w, False)
+        try:
+            for _ in range(20):
+                os.write(w, b"\x00")
+            daemon._drain_signal_pipe(r)
+            import pytest as _pytest
```

Ruff (0.15.11)

[warning] Incorrect import of `pytest`; use `import pytest` instead

(PT013)